### ISSUE TITLE
Trocar a fonte do site

### ISSUE THREAD

**Issue by GabrielCGon** [JoseOda/TrabalhoGCS#10]
Fazer a troca da fonte para Roboto para melhor experiência do usuário.

**Comment by hugor1256** [comment]
Boa ideia, irie fazer a alteração.

**Comment by JoseOda** [comment]
Concordo totalmente, a fonte antiga não é uma das melhores a serem utilizadas num website

**Comment by halls2001** [comment]
Acredito que essa fonte não ficou esteticamente legal, na minha opinão  deveremos trocar essa fonte, minha sugestão é trocarmos para a fonte Open Sans.
